projects/kit/components/input-phone-international/input-phone-international.style.less

```diff
@@ -0,0 +1,60 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+@flag-width: 1.75rem;
+@flag-height: 1.25rem;
+
+:host {
+    display: block;
+
+    &._disabled {
+        pointer-events: none;
+    }
+}
+
+.t-country-select {
+    width: 3.625rem;
+
+    & select:not(:disabled):not(._readonly) {
+        cursor: pointer;
+    }
+
+    &[data-size='m'] {
+        width: 5.5rem;
+    }
+
+    &[data-size='s'] {
+        width: 2rem;
+
+        .t-flag {
+            margin-left: -@space * 4;
+        }
+    }
+}
+
+.t-input-phone {
+    flex: 1;
+}
+
+.t-flag {
+    .fullsize(absolute, inset);
+
+    left: 0.5rem;
+    width: @flag-width;
+    height: @flag-height;
+    margin: auto;
+}
+
+.t-country-item-flag {
+    width: @flag-width;
+    height: @flag-height;
+}
+
+.t-country-item-name {
+    margin-left: @space * 3;
+    margin-right: auto;
+}
+
+.t-country-item-code {
+    color: var(--tui-text-02);
+    margin-right: @space;
```

Comment:
As a general rule of thumb we should try avoiding -left and -right for margin, padding and border and use -inline-start and -inline-end to be better prepared for RTL languages. It is now supported by all our browsers.

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/7580/commits/0b2f448326e9c9cf5a33becb91f000f7fba50525